frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -1651,9 +1658,27 @@ impl<T: Config> ElectionProvider for Pallet<T> {
 				Ok(supports)
 			},
 			Err(why) => {
-				log!(error, "Entering emergency mode: {:?}", why);
-				<CurrentPhase<T>>::put(Phase::Emergency);
-				Err(why)
+				// if `elect()` was called during `Phase::Off` or too early since the beginning of
+				// the signed phase, do not enter in emergency mode since there was no enough time
+				// to reach a solution.
+				match CurrentPhase::<T>::get() {
+					Phase::Off => {
+						Self::deposit_event(Event::EmergencyPhaseThrottled);
+						Err(why)
+					},
+					Phase::Signed(started_at)
```

Comment:
Will need small migration

Reply:
That's true, do you think it would be better to not touch `Phase` and add a new storage item that keeps the last block number that the signed phase was set?

Reply:
I think its a very simple migration so we should go with this. It could be something like this: 
- If at migration, the phase is anything other than `Signed` do nothing
- if phase is `Signed`, set block number to zero (implication is emergency phase won't be throttled till the next round). 

We could also try to estimate when the `Signed` phase would have started by reading the current election prediction but I think setting it to zero should be okay.